scripts/train.py

```diff
@@ -475,6 +664,30 @@ def main():
         val_loss = None
         test_loss = None
 
+    ## Update experiment configuration
+    exp_configure.update({"start_epoch": start_epoch})
+
+    ## Try and start wandb
+    try:
+        import wandb
```

Comment:
I think we would benefit from having this logic in a function and maybe it is better to have a CLI argument handle if we want to use `wandb` or not, defaulting to whatever is the expected sane behavior. Does that sound reasonable?